metricbeat/module/mongodb/dbstats/dbstats_integration_test.go

```diff
@@ -51,22 +51,24 @@ func TestFetch(t *testing.T) {
 		assert.True(t, collections > 0)
 
 		objects := metricsetFields["objects"].(int32)
-		assert.True(t, objects > 0)
+		assert.True(t, objects >= 0)
```

Comment:
This is because when there are no objects in database across all collections, it will show 0, which is why the change to accommodate either 0 or greater than 0.